cli/internal/cmdutil/errors.go

```diff
@@ -0,0 +1,14 @@
+package cmdutil
+
+// Error is a specific error that is returned by the command to specify the exit code
+type Error struct {
```

Comment:
`Error` is a pretty generic term. Maybe something like `ExitCodeError` instead? Or some other name to indicate what's special about this particular kind of error.